README.md

```diff
@@ -8,7 +8,7 @@ Full instructions for the setup and use of Run Scanner can be found in the [Run
 
 ## Prerequisites
 
-* JDK 11
+* JDK 17
 * Tomcat 8 or 9
 * C++ build environment (see [runscanner-llumina](runscanner-illumina/README.md))
 * [Maven 3.0.5](http://maven.apache.org/download.html) or later
```

Comment:
Oh, a Maven version this old will probably not play nice with Java 17? I am using 3.8.5, but is that OK to make the minimum or is there a version earlier than that I should specify?

Reply:
I think Maven 3.8+ is good

Reply:
Updated, along with CI